sql/catalyst/src/main/java/org/apache/spark/sql/connector/read/SupportsPushDownLimit.java

```diff
@@ -18,6 +18,7 @@
 package org.apache.spark.sql.connector.read;
 
 import org.apache.spark.annotation.Evolving;
+import org.apache.spark.sql.connector.expressions.SortValue;
```

Comment:
`SortValue` is `private[sql]`, we shouldn't expose it in the public APIs. We should use `SortOrder` instead